todo-app/src/components/AddingContainer.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+import moment from "moment"
+
+import FaPlus from "react-icons/lib/fa/plus"
+
+
+export default class AddingContainer extends Component {
+    state = {
+        description: "",
+        deadLine: ""
+    }
+    handleDescriptionChange = (event) => {
+        this.setState({ description: event.target.value })
+    }
+    handleDeadlineChange = (event) => {
+        this.setState({ deadLine: moment(event.target.value).format('ddd, DD MMM, YYYY') })
+
+    }
+    handleCreating = () => {
+        const descriptionInput = this.state.description
+        const deadLineInput = this.state.deadLine
+        if (descriptionInput && deadLineInput) {
+            this.props.onCreate(descriptionInput, deadLineInput)
+        } else {
+            alert("you should fill the values of description and deadline")
+        }    
+    }
+    handleSubmit = (e) => {
+        e.preventDefault()
+        this.handleCreating()
```

Comment:
There's no need here to create new function `handleCreating`. You can just add the logic of creating here.